maths/area.py

```diff
@@ -14,7 +14,7 @@ def surface_area_cube(side_length: Union[int, float]) -> float:
     >>> surface_area_cube(3)
     54
     """
-    return 6 * pow(side_length, 2)
+    return 6 * side_length ** 2

```

Comment:
Are we going to bounce back and forth every 48 hours?
https://github.com/TheAlgorithms/Python/pull/2223#pullrequestreview-453737047